blocks/url-input/index.js

```diff
@@ -180,11 +180,12 @@ class UrlInput extends Component {
 	render() {
 		const { value = '', instanceId } = this.props;
 		const { showSuggestions, posts, selectedSuggestion, loading } = this.state;
+		const autoFocus = 'undefined' === typeof this.props.autoFocus || this.props.autoFocus;
```

Comment:
This is a good use case for destructuring default assignment, just as we're doing with `value`:

```js
const { value = '', autoFocus = true, instanceId } = this.props;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Default_values_2

Reply:
@aduth Thanks.